WordPress/Classes/ViewRelated/Stats/Charts/Charts+AxisFormatters.swift

```diff
@@ -18,7 +18,7 @@ import Charts
 /// The workaround employed here (recommended in 2410 above) relies on the time series data being ordered, and simply
 /// transforms the adjusted values by the time interval associated with the first date in the series.
 ///
-class HorizontalAxisFormatter: IAxisValueFormatter {
+class HorizontalAxisFormatter: AxisValueFormatter {
```

Comment:
I was disappointed by the Charts releases on GitHub: I couldn't find an upgrade guide.

Luckily, GitHub can show us [the diff between the versions](https://github.com/danielgindi/Charts/compare/v3.6.0...v4.0.3). Add that to the browser's search feature and you can find a guide for these changes.